docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+### GitHub sunsets Atom
+
+<img src="./assets/sunsetting2.png" height="200" />
+
+So, having just mentioned how Microsoft had promised that Atom and VS Code would both see active development and support, it became clear that interest in Atom development was decreasing, new features weren't forthcoming, and issues were being left unresolved. The Atom.io blog page saw its last update in 2019 with the announcement of Atom 1.39, and most releases after that featured fixes and updates, but little in the way of exciting new features. The old Atom forum was shut down (but it is still browsable via the [Wayback Machine](https://web.archive.org/web/20210622052601/https://discuss.atom.io/)), and moved to the, then new, [GitHub Discussions](https://github.com/atom/atom/discussions) page, but there was little interaction from the original developers. The original package website was becoming unmaintained and hadn't had the "Featured" or "Trending" packages updated in a long time.
+
+Finally, on the 8th of June, 2022, GitHub made an announcement that Atom would be [sunset](https://github.blog/2022-06-08-sunsetting-atom/) in favour of VS Code and gave 6 months warning before the various services that make up the Atom ecosystem would go offline forever. The software itself and its source code would, of course, still be available via the archived repository, but the website, flight manual, package backend, and Teletype service would no longer be available.
+
+### Atom Community
+
+<img src="./assets/atom-community.png" height="200" />
+
+As soon as the news hit, there was a flurry of activity. Some were looking for direct replacements for Atom; others were looking for active forks or continuations of the project. [Atom Community](https://atom-community.github.io/) was the logical place for these discussions. This was a community that had taken on the maintenance of a bunch of Atom packages (originally for the Abandoned Facebook [Nuclide](https://nuclide.io/) project to turn Atom into an IDE) and had since taken on other packages from the original Atom maintainers.
+
+At this time, the Atom Community Discord server was rather active with discussions about what could be done. Could the community keep Atom alive? Could they take ownership of any of the original repos? Would a community project be allowed to use the original name? However, it became clear there was somewhat of a split in the goals of the group; some wanted to take Atom and radically change it, upgrade it and take it into the future, get it into a state it might have been in had development not slowed to a crawl. Others wanted to be a little more conservative, keep the Atom they knew and loved, and make sure it was kept up-to-date with patches and fixes.
```

Comment:
```suggestion
At this time, the Atom Community Discord server was rather active with discussions about what could be done. Could the community keep Atom alive? Could they take ownership of any of the original repos? Would a community project be allowed to use the original name? However, it became clear there was somewhat of a split in the goals of the group. Some wanted to take Atom and radically change it, upgrade it and take it into the future — bringing it into the state it might have been in had development not slowed to a crawl. Others wanted to be a little more conservative: keep the Atom they knew and loved and make sure it was kept up-to-date with patches and fixes.
```